### PR TITLE
A number of cleanup commits

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: benchmark/lib/report.dart [hunk 1]
```diff
@@ -36,14 +38,12 @@ pb.Platform createPlatform() {
...
 
 final bool _checkedMode = () {
-  bool x = true;
-  try {
-    // Trigger an exception if we're in checked mode.
-    x = "" as dynamic;
-    return x != ""; // return false; suppress unused variable warning
-  } catch (e) {
+  var checked = false;
+  assert(() {
```
Comment (rakudrama) [inline benchmark/lib/report.dart]: This is not checking the same thing.

The code as originally written:
 - in Dart 1 was checking for --checked.
 - in Dart 2 was checking for --omit-implicit-checks when compiled with dart2js

Now it is checking for --enable-asserts.

So I expect a performance regression on 'production' apps.

Comment (kevmoo) [reply benchmark/lib/report.dart]: @rakudrama – suggestions?

Comment (rakudrama) [reply benchmark/lib/report.dart]: revert this part

Comment (rakudrama) [reply benchmark/lib/report.dart]: And document it better.

**Pull request merged**